testsupport/tiers/checks.go

```diff
@@ -603,8 +653,9 @@ func userEditRoleBinding(userName string) spaceRoleObjectsCheck {
 		assert.Equal(t, "edit", rb.RoleRef.Name)
 		assert.Equal(t, "ClusterRole", rb.RoleRef.Kind)
 		assert.Equal(t, "rbac.authorization.k8s.io", rb.RoleRef.APIGroup)
-		assert.Equal(t, "codeready-toolchain", rb.ObjectMeta.Labels["toolchain.dev.openshift.com/provider"])
-		assert.Equal(t, owner, rb.ObjectMeta.Labels["toolchain.dev.openshift.com/owner"])
+		assert.Equal(t, toolchainv1alpha1.ProviderLabelValue, rb.ObjectMeta.Labels[toolchainv1alpha1.ProviderLabelKey])
+		assert.Equal(t, owner, rb.ObjectMeta.Labels[toolchainv1alpha1.OwnerLabelKey])
+		assert.Equal(t, owner, rb.ObjectMeta.Labels[toolchainv1alpha1.SpaceLabelKey])
```

Comment:
could these occurrences be replaced by `assertExpectedToolchainLabels` as well?

Reply:
I've missed those, thanks!

Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/726/commits/42199b69c34bf852ef4019bb74e6286bf62ff756